backend/src/main/java/ba/unsa/etf/si/bbqms/ws/models/BranchGroupCreateDto.java

```diff
@@ -0,0 +1,6 @@
+package ba.unsa.etf.si.bbqms.ws.models;
+
+import java.util.Set;
+
+public record BranchGroupCreateDto(String name, Set<Long> serviceIds, Set<Long> branchIds) {
```

Comment:
Is it easier for the frontend to send you id's, or entire objects? Discuss with @nejradilovic .